Tests/Fixtures/app/config/base_config.yml

```diff
@@ -15,6 +15,7 @@ bazinga_js_translation:
     active_locales:
         - fr
         - en
+        - it
```

Comment:
why adding italian translations in the testsuite ? this seems unnecessary to me.

Reply:
I added a new locale which would be _ICU-only_, but indeed I can keep `en` and `fr` locales only and add a new `messages-intl+icu.fr.yml` catalogue.